src/mono/sample/wasm/browser-threads-minimal/Program.cs

```diff
@@ -54,7 +54,7 @@ public static async Task TestCallSetTimeoutOnWorker()
             Console.WriteLine ($"XYZ: Main Thread caught task tid:{Thread.CurrentThread.ManagedThreadId}");
         }
 
-        const string fetchhelper = "./fetchelper.js";
+        const string fetchhelper = "../fetchhelper.js";
```

Comment:
While we're changing layouts and stuff, should we rename this file? dotnet.fetch-helper etc? Just a thought

Reply:
This is just MT sample which I will probably rewrite multiple times yet. Don't worry about it.